shared/build.gradle.kts

```diff
@@ -93,3 +120,41 @@ android {
 tasks.check {
     dependsOn(detekt)
 }
+
+buildkonfig {
+    packageName = "co.nimblehq.blisskmmic"
+
+    defaultConfigs {
+        buildConfigField(
+            STRING,
+            "CLIENT_ID",
+            BuildKonfig.CLIENT_ID
+        )
+        buildConfigField(
+            STRING,
+            "CLIENT_SECRET",
+            BuildKonfig.CLIENT_SECRET
+        )
+        buildConfigField(
+            STRING,
+            "BASE_URL",
+            BuildKonfig.STAGING_BASE_URL
+        )
+    }
+
+    defaultConfigs("production") {
+        buildConfigField(
+            STRING,
+            "BASE_URL",
+            BuildKonfig.PRODUCTION_BASE_URL
+        )
+    }
+
+    defaultConfigs("staging") {
+        buildConfigField(
+            STRING,
+            "BASE_URL",
+            BuildKonfig.STAGING_BASE_URL
+        )
```

Comment:
Having `TODO` for this part could be a helpful reminder for you to work on providing correct values later on 🙏

Reply:
This config file is provided in Github Secrets and provided locally with gitignore 🙏